keyboards/skyloong/gk68/pro/config.h

```diff
@@ -0,0 +1,11 @@
+// Copyright 2023 linlin012 (@linlin012)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define AW20216S_EN_PIN_1 B5
+#define AW20216S_CS_PIN_1 B12
+#define CAPS_LOCK_INDEX 30
+#define WIN_MOD_INDEX 17
+#define MAC_MOD_INDEX 18
+#define SDB B5
```

Comment:
This is a redefine of `AW20216S_EN_PIN_1`, and should be used instead. 

Could you update all of the instances of these? 
```suggestion
```